packages/components-v2/src/support/types/index.ts

```diff
@@ -1,6 +1,6 @@
 import { COINS, ICONS } from '../../config/variables';
 
-export type FeedbackType = 'success' | 'info' | 'warning' | 'error';
+export type FeedbackType = 'success' | 'warning' | 'error';
```

Comment:
not used anywhere in the input, and we are going to need info notifications anyway.